### ISSUE TITLE
Publish image filename and acquisition date in pdf report

### ISSUE THREAD

**Issue by alanphys** [jrkerns/pylinac#499]
It would be helpful to publish the image filenames and acquisition dates in the pdf report.

A workaround may be to put this in the metadata.

Regards
Alan

**Comment by jrkerns** [comment]
Ah, you mean of the original files? That makes sense if they are DICOM.

**Comment by alanphys** [comment]
Yes, I listed the issue from this [response](https://forum.pylinac.com/t/gui-frontend-to-pylinac-and-pydicom/413/5?u=alanphys)  on the Pylinac forum.

I see some of the images e.g. DRGS and DRMLC have image filenames. For Catphan and Winston-Lutz you may just have to give the base folder.
